src/main/java/org/c4sg/controller/SkillController.java

```diff
@@ -47,7 +50,39 @@
     @RequestMapping(value="/project",produces = {"application/json"}, method = RequestMethod.GET)
     @ApiOperation(value = "Get skills for a project by id", notes = "Returns a collection of skills for a project")
     public List<String> getSkillsForProject(@ApiParam(value = "ID of project to return", required = true)
-    									 @RequestParam Integer id) {
+    									 	@RequestParam Integer id) {
         return skillService.findSkillsForProject(id);
     }
+    
+    @CrossOrigin
+    @RequestMapping(value="/user/save", method = RequestMethod.POST)
```

Comment:
@felipealves-gnu I should have been more careful in choosing the name. I will remove it. Thanks.